README.md

```diff
@@ -4,7 +4,9 @@
 
 ## Requirements
 
-All the requirements for building the library (in `./lib`) are fetched automatically with CMake.
+Building and installing the csnet library requires CMake 3.22 or newer.
+
+All dependencies of the library (in `./lib`) are fetched automatically with CMake.
```

Comment:
This did not install `doxygen`,  I had to install doxygen manually.

Reply:
Yeah, maybe I should change the configuration command in the docs to:
```
cmake -DBUILD_EXAMPLES=ON -DBUILD_TESTS=OFF -DBUILD_DOCS=OFF -B dist
```
such that the docs are not built by default. Doxygen is actually listed as a requirement in the Development chapter of the README. I made this more clear in #25.